### PR TITLE
Css addquestion

### PR DESCRIPTION
**Pull request by MorganeL-pro**
CSS for views "add-question", "result" and "success"
Added confettis with JS if there's a good answer.

### CODE REVIEW

**Review by Benjamin-51 — DISMISSED** [review]
Works well on my end.

File: src/View/Question/success.html.twig [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+        if (timeLeft <= 0) {
+            return clearInterval(interval);
+        }
+
+        var particleCount = 50 * (timeLeft / duration);
+        // since particles fall down, start a bit higher than random
+        confetti(Object.assign({}, defaults, { particleCount, origin: { x: randomInRange(0.1, 0.3), y: Math.random() - 0.2 } }));
+        confetti(Object.assign({}, defaults, { particleCount, origin: { x: randomInRange(0.7, 0.9), y: Math.random() - 0.2 } }));
+        }, 250);
+    </script>
```
Comment (RomainClair) [inline src/View/Question/success.html.twig]: You should have your script in a .js file

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -9,10 +9,10 @@
     <title>Quizzy - le plus grand quiz du monde !{% block title %}{% endblock %}</title>
 
     <link rel="stylesheet" href="/assets/css/style.css">
-    <link rel="stylesheet" href="/assets/css/formDisplayQuestion.css">
     {% block stylesheet %}{% endblock %}
     <link rel="icon" href="/assets/images/favicon.png">
     <script src="https://kit.fontawesome.com/ab41dec404.js" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/canvas-confetti@1.4.0/dist/confetti.browser.min.js"></script>
```
Comment (RomainClair) [inline src/View/layout.html.twig:15]: You could consider to host you assets locally.
One of the benefit would be to have your site working offline

**Review by RomainClair — CHANGES_REQUESTED** [review]

**Comment by MorganeL-pro** [comment]
I cannot put some scripts in a .js files because of twig variables.
trying to have a fontawesome file in the project but not working yet

**Review by Benjamin-51 — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**